android/KMEA/app/build.gradle

```diff
@@ -42,25 +42,40 @@ android {
                 //showStandardStreams = true // If we want test console log output to show during builds.
             }
             workingDir = "../" // Defaults to the `app` subdirectory, which is different from Android Studio's default.
+            //Avoid Warning: an illegal reflective access operation has occurred
```

Comment:
Why do we get this warning? Can you explain in the comment please

Reply:
It is a problem in robolectric.

Reply:
Ok, please add a comment to note that it relates to robolectric (and if possible a URL to a relevant report against that project)

Reply:
https://github.com/robolectric/robolectric/issues/4776

Reply:
I'm sorry, I meant a code comment -- because we want to be able to see the reference in the code later without coming back to the PR

Reply:
Reading that issue report, I see that `WARNING: All illegal access operations will be denied in a future release` which means that simply disabling the warning is just going to cause us trouble.

Reply:
In java 11  you can just ignore this warning and it will work. 
When they will deny this illegal accesses you need to add these `jvmArgs('--add-opens', 'java.base/java.lang=ALL-UNNAMED')` for all packages you may use.
So I fix the illegal access warnings. See also my comments on Issue #2243